docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -1084,7 +1084,38 @@
     "jobs" : {
       "type" : "array",
       "items" : {
-        "type" : "any"
```

Comment:
The non-openapi docs don't further analyze objects if they use a custom serializer, hence why this was basically empty before.